src/main/java/picard/vcf/SplitVcfs.java

```diff
@@ -73,16 +73,26 @@ public SplitVcfs() {
 
     @Override
     protected int doWork() {
-        IOUtil.assertFileIsReadable(INPUT);
+        Path inputPath;
+        try {
+            inputPath = IOUtil.getPath(INPUT);
+        } catch (IOException e) {
+            throw new RuntimeIOException(e);
+        }
+        IOUtil.assertPathsAreReadable(Stream.of(inputPath).collect(Collectors.toList()));
```

Comment:
there's only one path here, no? why the stream and list?

Reply:
There is no `assertPathIsReadable(Path p)` in `IOUtils` so I have to make a `List`, however I can use the more concise `Collections.singletonList(inputPath)`